### DISCUSSION TITLE
Release request for 25.0.6

### DISCUSSION THREAD

**Discussion by austinvazquez** [moby/moby#48200] [Q&A] [answered]
I wanted to see if we can get a new cut of 25.0.

**Comment by thaJeztah** [comment]
I think it's fine to do a tag; @corhere any objections? (if not, feel free to do a tag)

@austinvazquez do you need a new tag on the CLI as well? I think there's some backports that haven't been done yet; if you're interested; feel free to open PRs for those

https://github.com/docker/cli/pulls?q=is%3Apr+label%3Aprocess%2Fcherry-pick%2F25.0+is%3Aclosed

**Reply by austinvazquez** [reply]
Thanks for taking a look @thaJeztah . I'll head over to CLI and see if I can help get those pushed through. :)

**Comment by swagatbora90** [comment]
@austinvazquez Thanks for the awesome work here. Do we have a path forward on this release?

**Reply by austinvazquez** [reply]
I have worked to push through the items in Docker CLI for the 25.0.6 milestone. @corhere, any objection for a 25.0.6 tag? I'm happy to help if there is anything I can do. 
https://github.com/docker/cli/milestone/107

**Reply by corhere** [reply]
No objection from me

**Comment by corhere** [comment, marked as answer]
https://github.com/moby/moby/releases/tag/v25.0.6

**Reply by austinvazquez** [reply]
Thanks @corhere , @thaJeztah for the help here.
